spec/system/task_spec.rb

```diff
@@ -1,24 +1,27 @@
 require 'rails_helper'
+include ApplicationHelper
 
 RSpec.describe 'Task', type: :system do
   describe 'Task一覧' do
+    let(:project) {create(:project)}
+    let(:task) {create(:task, project_id: project.id)}
+
     context '正常系' do
       it '一覧ページにアクセスした場合、Taskが表示されること' do
         # TODO: ローカル変数ではなく let を使用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
         visit project_tasks_path(project)
         expect(page).to have_content task.title
         expect(Task.count).to eq 1
         expect(current_path).to eq project_tasks_path(project)
       end
 
-      xit 'Project詳細からTask一覧ページにアクセスした場合、Taskが表示されること' do
+      it 'Project詳細からTask一覧ページにアクセスした場合、Taskが表示されること' do
         # FIXME: テストが失敗するので修正してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
         visit project_path(project)
         click_link 'View Todos'
+        windows = page.driver.browser.window_handles
+        page.driver.browser.switch_to.window(windows.last)
+        
```

Comment:
１行で記載できることは知っておいてください。また余計な改行は消した方が良いでしょう。
```suggestion
switch_to_window(windows.last)
```

Reply:
3e38ac9f41c853c08fb4de09db7c201f0174717e
コードを一行で記載